File: LeetCode/344_ReverseString.cc
// https://leetcode.com/problems/reverse-string/
// 2022年04月01日 21:55:48

class Solution {
public:
    void reverseString(vector<char>& s) {
        int i=0,j=s.size()-1;
        char temp;
        while(j>i){
            temp=s[i];
            s[i]=s[j];
            s[j]=temp;
            i++;
            j--;
        }
        
    }
};

File: LeetCode/344_ReverseString.go
// https://leetcode.com/problems/reverse-string/
// 2022年07月05日 12:34:42

func reverseString(s []byte)  {
    for i,j:=0,len(s)-1;i<j;i,j=i+1,j-1 {
        s[i],s[j]=s[j],s[i]
    }
}